### PR TITLE
doc/governance: add exec council responsibilites

### PR DESCRIPTION
**Pull request by zdover23**
Add the Ceph Executive Council's responsibilties to the doc/governance.rst document. It was decided during the weekly CLT meeting on 30 Sep 2024 to add this to the ceph/ceph git repository.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/governance.rst [hunk 1]
```diff
@@ -21,14 +21,56 @@ Bodies
...
-Responsibilities
-----------------
+.. _exec-council-responsibilities:
+
+Ceph Executive Council Responsibilities
+---------------------------------------
+
+- Spokesperson
+
+   - welcome/keynote for cephalocon
```
Comment (anthonyeleven) [inline doc/governance.rst:31]: Suggest capitalizing the bullets, and of course Cephalocon

File: doc/governance.rst [hunk 2]
```diff
@@ -21,14 +21,56 @@ Bodies
...
+   
+- Community
+
+    - managing community manager
+
+    - LF Program Manager person, Social Media person
+
+    - liase with the ambassadors
+
+    - make sure ceph events happen, successfully: cephalocon, ceph days, cds, user/dev, cdm
```
Comment (anthonyeleven) [inline doc/governance.rst:46]: caps

File: doc/governance.rst [hunk 3]
```diff
@@ -21,14 +21,56 @@ Bodies
...
+
+    - coordinating with LF
+
+    - creating program committee
+
+    - recordings on youtube
```
Comment (anthonyeleven) [inline doc/governance.rst:52]: caps

**Review by anthonyeleven — APPROVED** [review]
Caps suggestions, approved in advance for streamlining

**Comment by zdover23** [comment]
> Caps suggestions, approved in advance for streamlining

I'm going to add these in a follow-up. I just wanted to get this PR in so that I didn't forget it. I'm juggling six projects right now. This kind of juggling is currently necessary.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/60139 - Squid backport
https://github.com/ceph/ceph/pull/60140 - Reef backport
https://github.com/ceph/ceph/pull/60141 - Quincy backport

File: doc/governance.rst (hunk 1 above)
Comment (zdover23) [reply doc/governance.rst:31]: https://github.com/ceph/ceph/pull/60142

File: doc/governance.rst [hunk 4]
```diff
@@ -21,14 +21,56 @@ Bodies
...
+
+    - ensure foundation is healthy: financials, operations
+
+    - represent the CLT on the Board
+
+    - present project status regularly (yearly)
+
+    - collect member ideas / feedback
+
+    - ensure members feel valued
```
Comment (batrick) [inline doc/governance.rst:71]: no no, we don't edit this doc except by consensus in the CLT meeting. This is our governance document (like a constitution).

The additions here should go in another doc that lists "responsibilities in practice". Alternatively, we can put this PR up for a vote.

Comment (zdover23) [reply doc/governance.rst:71]: @batrick 

I'll roll this back, and we can have a short chat in the CLT in two-and-a-half hours, and we'll get it right.

**Pull request merged**